cmd/soroban-cli/.soroban/identities/test_id.toml

```diff
@@ -0,0 +1 @@
+secret_key = "SBGWSG6BTNCKCOB3DIFBGCVMUPQFYPA2G4O34RMTB343OYPXU5DJDVMN"
```

Comment:
This is probably fine for now, but I'd say tests should generate a random strkey with `rs-stellar-strkey`, and put it into a toml file in a tmpdir.

Reply:
Oh.. We need this for the *other* tests. And there's no default `identity` so they need a key to run.. 🤔 Still feels like a bit of a kludge...

Reply:
See above commend to @tsachiherman, but now the tests each have their own temp directory and identities are written when a test needs one.